fuzz/fuzz_targets/differential-new.rs

```diff
@@ -0,0 +1,85 @@
+#![no_main]
+
+use libfuzzer_sys::arbitrary::{Result, Unstructured};
+use libfuzzer_sys::fuzz_target;
+use wasmtime_fuzzing::generators::{DiffValue, SingleInstModule};
+use wasmtime_fuzzing::oracles::engine::{get_exported_function_signatures, DiffEngine};
+use wasmtime_fuzzing::oracles::{diff_wasmtime, differential, engine};
+
+const NUM_INVOCATIONS: usize = 5;
+
+fuzz_target!(|data: [u8; 1024]| {
```

Comment:
I think if this is `&[u8]` instead of `[u8; 1024]` it's whatever the fuzzer gives, and otherwise it's required to be 1kb?